controller/lib/core/nvparam.cpp

```diff
@@ -0,0 +1,218 @@
+/* Copyright 2020, RespiraWorks
```

Comment:
I notice that the filename is nvparam, but the function names are nvparams; is that intentional?